public/locales/en/translation.json

```diff
@@ -365,7 +365,8 @@
       "updatePermissions": "Some scenario permissions have not been updated correctly.",
       "run": "A problem occurred during the scenario run.",
       "update": "A problem occurred during scenario update; your new parameters haven't been saved.",
-      "incompleteRun": "A problem occurred during dataset update; the scenario is running but your new parameters haven't been saved.",
+      "incompleteRun": "A problem occurred during dataset update; your new parameters haven't been saved.",
+      "datasetUpdateWithLaunch": "A problem occurred during dataset update; new parameters haven't been saved and launch has been canceled.",
```

Comment:
The key name `incompleteRun` is no longer consistent with the message (that doesn't mention scenario runs), whereas your second label mentions the scenario launch. Maybe we should switch the labels and change the second key to `datasetUpdate`?